spec-dtslint/operators/filter-spec.ts

```diff
@@ -51,7 +51,7 @@ it('should support Boolean as a predicate', () => {
 // https://github.com/ReactiveX/rxjs/issues/4959#issuecomment-520629091
 it('should support inference from a return type with Boolean as a predicate', () => {
   interface I {
-    a: string | null;
+    a: string | null | undefined | false | 0 | -0 | 0n | '';
```

Comment:
I think it would be best to not make **any changes** to this test. It's specifically about a comment that Rado made regarding an inference quirk - that no longer seems to happen, so it could have been a TypeScript bug.

Instead, add more expectations - for the different falsy types - to the test that's above this one.

Reply:
> Instead, add more expectations - for the different falsy types - to the test that's above this one.

Yep. That makes a lot of sense. I will do that, for sure. Thanks!

Reply:
Done, thanks!